vite.config.js

```
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  build: {
```

ESLint

[error] Insert `··`

(prettier/prettier)